pom.xml

```diff
@@ -20,7 +20,7 @@
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
         <cloudevents.version>2.3.0</cloudevents.version>
-        <spring-cloud.version>2021.0.2</spring-cloud.version>
+        <spring-cloud.version>2021.0.1</spring-cloud.version>
```

Comment:
Spring Native 0.11.4 has been tested against Spring Cloud 2021.0.1.

Reply:
Did you find any issues? I can see other spring boot projects being updated recently to these versions:
https://github.com/knative-sandbox/kn-plugin-func/pull/999/files